### PR TITLE
Fix groovy builds

### PR DESCRIPTION
**Pull request by jackpot51**

### CODE REVIEW

**Review by ids1024 — APPROVED** [review]

**Comment by jackpot51** [comment]
Clearly something isn't right. I'll force push when it is fixed

**Comment by jackpot51** [comment]
Ok, it's fixed now

File: debian/control [hunk 1]
```diff
@@ -7,6 +7,7 @@ Build-Depends:
   cargo,
   rustc (>=1.36.0),
   libgtk-3-dev,
+  libxml2-dev,
```
Comment (ids1024) [inline debian/control:10]: This should be avoidable by using `default-features = false` with the gtk-extras dependency, if the svg feature there isn't needed.

Perhaps that feature shouldn't be default...

Comment (ids1024) [reply debian/control:10]: Though it may not matter, really. While it adds a dev dependency, gtk probably pulls in libxml at runtime anyway.

**Review by jacobgkau — APPROVED** [review]
This widget's toggles in Settings -> Displays still work as expected for controlling the HiDPI daemon (on Focal) after the change.

**Pull request merged**